### PR TITLE
docs: Update the readme to add dynamic lora weights loading on Docker & BentoCloud

### PR DESCRIPTION
**Pull request by Sherlock113**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -216,9 +216,39 @@ onediffusion build stable-diffusion
...
+onediffusion build stable-diffusion-xl --lora-dir "/path/to/lorafiles/dir/"
+```
+
+If you only have a single LoRA file to use, run the following instead:
+
+```bash
+onediffusion build stable-diffusion-xl --lora-weights "/path/to/lorafile"
+```
+
+Each Bento has a randomly created `BENTO_TAG` as its unique identifier. To customize it, use `--name` and `--version` options.
```
Comment (larme) [inline README.md:233]: The bento tag's name is construct from model_name, model_id and other factors. the tag's version is also not randomly generated but match model's version. The model's tag is followed huggingface's commit hash. That's why the same command will generally generate the same tag. Hence I think here we should not use "randomly". Can we say that by default each bento will generate a name containing information like model name, model_id etc.?

**Review by larme — CHANGES_REQUESTED** [review]
There's only the wording about bento tag need to be change

File: README.md (hunk 1 above)
Comment (Sherlock113) [reply README.md]: Thanks. Changed.

**Review by larme — APPROVED** [review]
LGTM

**Pull request merged**